dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -414,6 +414,16 @@ If you have providers as list of provider ids because you just released them you
 ```shell script
 cd "${AIRFLOW_REPO_ROOT}"
 
+breeze release-management publish-docs providers-index amazon cncf.kubernetes --override-versioned --run-in-parallel
+
+breeze release-management add-back-references all-providers
+```
+
+or with
+
+```shell script
+cd "${AIRFLOW_REPO_ROOT}"
```

Comment:
Is this missing some command later? It ends after CD, is this intentional?

Reply:
It doesn't end with cd the next command is executing a script
Open the file in the new version to see it.